projects/schematics/src/migrations/2_0/ng-add-localize/ng-add-localize.ts

```diff
@@ -46,11 +49,23 @@ function installPackageJsonDependencies(): Rule {
 }
 
 export function migrate(): Rule {
-  return () => {
-    return chain([
-      addPackageJsonDependencies(),
-      installPackageJsonDependencies(),
-      externalSchematic(ANGULAR_LOCALIZE, 'ng-add', {}),
-    ]);
+  return (tree: Tree, context: SchematicContext) => {
+    const angularLocalizeInstalled = isAngularLocalizeInstalled(tree);
+    if (angularLocalizeInstalled) {
+      context.logger.info(
+        `Skipping the installation of ${ANGULAR_LOCALIZE} as it's already installed.`
+      );
+    } else {
+      context.logger.warn(
+        `Please run the following: ng add ${ANGULAR_LOCALIZE}`
+      );
+    }
+
+    return angularLocalizeInstalled
+      ? noop()
+      : chain([addPackageJsonDependencies(), installPackageJsonDependencies()])(
+          tree,
+          context
+        );
```

Comment:
Not sure if we need to provide `(tree, context)`. From my experience, it works just fine without providing those.